### ISSUE TITLE
legacy resolver not compatible

### ISSUE THREAD

**Issue by olsonpm** [un-ts/eslint-plugin-import-x#424]
From what I can tell, [eslint-import-resolver-alias](https://github.com/johvin/eslint-import-resolver-alias) should be compatible with this plugin.  [This minimal reproduction](https://github.com/olsonpm/repro/tree/eslint-plugin-import-x-alias) shows it not working.

e.g.

```sh
$ git clone git@github.com:olsonpm/repro.git
$ cd repro
$ git checkout eslint-plugin-import-x-alias
$ npm ci

# eslint-plugin-import works
$ npm run lint
> eslint -c eslint.config-import.mjs dir/a.mjs

# but not eslint-plugin-import-x
$ npm run lint-x
> eslint -c eslint.config-import-x.mjs dir/a.mjs

/path/to/repro/dir/a.mjs
  1:15  error  Unable to resolve path to module '@/b.mjs'  import-x/no-unresolved

✖ 1 problem (1 error, 0 warnings)
```

---

Thanks for forking eslint-plugin-import.  I had a lot of ugly workarounds that are no longer necessary.

**Comment by JounQin** [comment]
The setting key should be pfefixed with `import-x` not only `import`.

**Comment by olsonpm** [comment]
wow I'm sorry for missing that
